Tapawingo_backend/Controllers/OrganisationsController.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Tapawingo_backend.Dtos;
+using Tapawingo_backend.Services;
+
+namespace Tapawingo_backend.Controllers
+{
+    [ApiController]
+    [Route("api/[controller]")]
+    public class OrganisationsController : ControllerBase
+    {
+        private readonly OrganisationsService _organisationsService;
+
+        public OrganisationsController(OrganisationsService organisationsService)
+        {
+            _organisationsService = organisationsService;
+        }
+
+        //TODO: ADD AUTHORIZATION RULE
+        [HttpGet("{id}")]
+        [ProducesResponseType(StatusCodes.Status200OK, Type = typeof(OrganisationDto))]
+        [ProducesResponseType(StatusCodes.Status404NotFound)]
+        public IActionResult GetOrganisation(int id)
+        {
+            var organisation = _organisationsService.GetOrganisationById(id);
+            return organisation!=null ? 
+                Ok(organisation) : 
+                NotFound("Organisation with this id was not found.");
+        }
+
+        //TODO: ADD AUTHORIZATION RULE
+        [HttpPost]
+        [ProducesResponseType(StatusCodes.Status201Created, Type = typeof(OrganisationDto))]
+        [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+        [ProducesResponseType(StatusCodes.Status400BadRequest)]
+        public IActionResult CreateOrganisation([FromBody]CreateOrganisationDto name)
```

Comment:
Hoort deze wel in dit merge request? Deze staat nu ook in https://github.com/luuk-royal/tapawingohike_backend/pull/4

Zijn nog meer dingen die dubbel lijken, als dit geen problemen oplevert dan kan deze ook approved en gemergd worden